types/hero.ts

```diff
@@ -113,15 +113,40 @@ export interface Equipment {
   name: string;
   slot: EquipmentSlot;
   type: EquipmentType;
+  effect: EquipmentEffect;
+  stat1: EquipmentStat | null;
+  stat2: EquipmentStat | null;
+  notes: string;
+}
+
+export interface ExclusiveEquipment {
+  name: string;
+  slot: string;
   effect: string;
-  stat1: string | null;
-  stat2: string | null;
+}
+
+export interface EquipmentStat {
+  type: string;
+  lvl1: string;
+  lvl10: string;
+  lvl20: string;
+  lvl30: string;
+  lvl40: string;
+  lvl50: string;
+}
+export interface EquipmentEffect {
+  lvl1: string;
+  lvl10: string;
+  lvl20: string;
+  lvl30: string;
+  lvl40: string;
+  lvl50: string;
```

Comment:
Would just  making an `EquipmentLevelBased` interface and having these two extend from that make sense here?

Reply:
i think so. But im not sure about this:
EquipmentEffect extends EquipmentLevelBased {} 

should i just use EquipmentLevelBased